melodic/distribution.yaml

```diff
@@ -6184,6 +6184,20 @@ repositories:
       url: https://github.com/milvusrobotics/mrp2_common.git
       version: melodic-devel
     status: maintained
+  mrp2_desktop:
+    doc:
+      type: git
+      url: https://github.com/milvusrobotics/mrp2_desktop.git
+      version: melodic-devel
+    release:
+      packages:
+      - mrp2_desktop
+      - mrp2_viz
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/milvusrobotics/mrp2_desktop-release.git
+      version: 0.2.2-1
+    status: maintained
```

Comment:
In Indigo, this had a `source` block.  It's not required for merging, but would you like to add it here as well?

Reply:
@OnurDz Friendly ping 🙂

Reply:
Sorry for late response. Closing the request for a new one.